src/sentry/rules/conditions/high_priority_issue.py

```diff
@@ -33,6 +33,15 @@ def passes(self, event: GroupEvent, state: EventState) -> bool:
         if not has_high_priority_issue_alerts(self.project):
             return False
 
+        if not self.rule:
+            return state.is_new
+
+        if not self.rule.data.get("new_issue_threshold_met"):
```

Comment:
If I were coming across this code for the first time I'd appreciate a comment explaining how this state gets set.

Reply:
This feels like it should be a project level thing?

Reply:
yeah - I didn't see a metadata-esque field in Project where this made sense. I _could_ add it as a flag?